undo/FontColorStep.cpp

```diff
@@ -4,7 +4,7 @@
 #include "FontColorStep.h"
 #include "ui_MainWindow.h"
 
-FontColorStep::FontColorStep(MainWindow* window, QColor& origC, QColor& newC) : UndoStep(window)
+FontColorStep::FontColorStep(MainWindow* window, QColor const & origC, QColor const & newC) : UndoStep(window)
```

Comment:
Good catch on the const; I need to go through and audit this project sometime to use const references instead of copying classes everywhere.